packages/falso/src/lib/random-uuid.ts

```diff
@@ -1,106 +1,7 @@
-    'ee3fa3e4-c0fb-4636-bd4c-e7b39d581b24',
-    'f8d53362-a1c7-41c2-a4b6-e186748ff1db',
-    '920e2944-e742-4d63-9b96-8311a42d0972',
-    '5540f00f-bd2d-4498-9443-cf73d25493e3',
-    'aeff09d2-a30a-4a8c-a10f-ca82339c56c1',
-    'eda4158b-c4ff-4f65-97da-ef9e8e8ac700',
-    '7ea8849d-494b-4244-b492-8ff266d7264c',
-    '59815243-bc11-43c3-a447-033fcac6a5ff',
-    '6fb54d4c-5630-4f0e-b623-fe51456875b8',
-    '062a2d76-9063-4067-b7b1-b38612b0ccdd',
-    '98c88158-d43e-4d05-82ee-e78a5249150f',
-    'dd5d5c2d-e084-4049-a8b6-de63a3bb14f0',
-    'a421d6c5-504d-4a2c-b6b0-f8402b00f894',
-    'b5f258ea-139e-43f1-86dd-39486e8674a4',
-    '2dff99a3-d6e0-4036-bac1-9817b53fe8d2',
-    'eed5f38a-8816-4ef0-9a22-177a48c1e811',
-    '4d2aaf84-ceed-4786-9c1e-310d776933f4',
-    'd5e44bdf-fc3f-49d5-ac6c-9a948423b5cf',
-    'b7f73d18-dad2-4189-953d-4a19fce8bc3e',
-    'c0b0bf4b-6607-4737-bd43-45e72d6471d5',
-    '3755f224-889d-47bc-b747-d4624a58bb8b',
-    '62ac66b1-6822-4230-9f88-c323d9699095',
-    '3ef7f13e-79dd-4178-bb55-3a9a8a59473f',
-    '0e25cbc6-a919-4be4-8aab-498a209bce92',
-    '9b8204b4-5863-4669-8ca0-de77fe07bbe6',
-    'd8a6952f-6b0e-4f8f-b86f-930cfb271fc4',
-    'e9fc3402-c361-430f-9f28-bd4b258939bd',
-    'a862c64a-a2a4-4cb2-b543-46836d07783c',
-    'c7161ab9-8a69-4c38-ad98-fa939fa5f6f1',
-    '6ff0dfc5-ff47-458d-963a-cd3534257cad',
-    'a5dd9958-2e8a-4927-ba05-39244b0d55cc',
-    '47183f7b-5a2f-4882-b185-afd0fd2e442f',
-    '7e25db59-f983-47d3-947e-35e0cd9a0ef6',
-    '6074563e-80aa-4610-9f3c-e95b08bacfe3',
-    'e12b727f-a44b-4f1b-b6c2-ef6d6eb70f4f',
-    'ce5dfa03-6797-4570-be32-23e7e3b7ba89',
-    '6e4b102f-6c9a-496d-8c77-734f38076767',
-    '09fb0437-5702-4ed0-91cd-9111b25689df',
-    'c7f6541e-8974-44e0-8f6a-e871cdd7f45d',
-    '89b4fa56-2376-475e-8ad8-1c052b83fb96',
-    '17b948c1-6b61-4ee6-83e4-97b8ad54231a',
-    'd35cbab8-ef2c-4fa4-bf91-308449ee766f',
-    '79f0362b-d9ad-48f7-a789-1e9eaf807c85',
-    '753d0159-67ed-44c7-aec9-8ffd4faa470b',
-    'bd0da628-a4db-4110-b5e8-6de935fd9426',
-    'c75bcc26-c030-4132-9311-11675bea79a8',
-    '67d05016-07ac-4804-8d83-2ec18712a9c6',
-    '85e28b60-f177-43db-b6fa-3720b22e03dd',
-    '265c5948-cf96-4b3f-b6a2-90d37e74e019',
-    '65ca84f8-a9ae-4d0a-b472-691ab43342da',
-    '07844055-4449-4e95-aab5-baae7d4b381c',
-    '546fb6bc-f0da-4ef8-aff4-de43681adda3',
-    '38567f6d-21a9-454c-a2ab-2c2d33c2e124',
-    '81d48333-1cc3-470c-9cab-c9065288d55d',
-    '5994c1a8-4517-432d-8ffb-8ad251a312ce',
-    '8ca3de45-2a4b-4f85-9002-d75748971e02',
-    '3e4f747c-ad9f-4345-9c70-ed112dd60b34',
-    '319c243d-2908-4f48-8d00-bc7b2fd475f2',
-    '233d6d98-6d6a-4bb6-97f3-90a467fbd0a5',
-    '7591d3d9-0d36-4631-9257-5d5e03746e34',
-    'e982c2a1-37ea-4d4c-bf41-f9a3b033b10a',
-    'd86c9da0-9e66-46c0-b88b-b47f0a231494',
-    'a432864a-85f7-4202-9312-b34590c6fb63',
-  ]);
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Random valid UUID
+ */
+export function randomUuid(): string {
+  return uuidv4();
```

Comment:
The library is not small. Can you check if it's tree-shakable and only pulls the code relevant to this function?

Reply:
I saw in bundlephobia, and they say that `uuid` is tree-shakable. 
![image](https://user-images.githubusercontent.com/2914170/149012900-b8bc7530-3fca-444a-b068-b069e0282b86.png)

Reply:
Cool, so we don't need the specs.

Reply:
As mentioned by @raisiqueira the lib is tree-shakable and the impact of an individual import is quite small. Note nx's report:

**WITH:**
Bundling falso...
  index.esm.js 824.486 KB
  index.cjs.js 828.891 KB
⚡ Done in 20.35s

**WITHOUT**
Bundling falso...
  index.esm.js 824.456 KB
  index.cjs.js 828.855 KB
⚡ Done in 11.60s